include/interface/drivers/QDAC.h

```diff
@@ -0,0 +1,33 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <span>
+
+namespace leka {
+
+enum class Channel : uint8_t
+{
+	A,
+	B,
+	C,
+	D
+};
+
+namespace interface {
+	class QDAC
+	{
+	  public:
+		virtual ~QDAC() = default;
+
+		virtual void init()														= 0;
+		virtual void write(Channel channel, uint16_t data, bool eeprom = false) = 0;
+		virtual auto read(Channel channel, bool eeprom = false) -> uint16_t		= 0;
```

Comment:
Merci de confirmer si c'est bien ces actions qui en découle selon la valeur booléenne~
```suggestion
		virtual void write(Channel channel, uint16_t data, bool save_in_eeprom = false) = 0;
		virtual auto read(Channel channel, bool load_from_eeprom = false) -> uint16_t		= 0;
```